### PR TITLE
Fixes scrolling lag on reporting table

### PR DESCRIPTION
**Pull request by bkovacik**
Closes KoddiDev/HPA#6736.

I added a scroll method to a new default container. It falls back to the previous behavior if one is not provided, so it shouldn't break any other code that relies on this.

Also softened tabs.

### CODE REVIEW

File: dist/reporttable.js [hunk 1]
```diff
@@ -17,229 +17,238 @@
...
+        this.reset();
+        
+        this.refreshFunct = $.proxy(this.scroll, this);
+        this.resizeFunct = $.proxy(this.resize, this);
+        
+        $(window).scroll(this.refreshFunct);        
+        $(window).resize(this.resizeFunct);
+    
+        var scrollDefaultObject;
+        console.log(this.options.scrollDefaultObject);
```
Comment (kyrozetera) [inline dist/reporttable.js]: A couple of `console.log`s got left behind.

File: dist/reporttable.js [hunk 2]
```diff
@@ -17,229 +17,238 @@
...
+    
+    ReportTable.prototype.defaultOptions = {
+        offsetHeaderHeight:0,
+        offsetFooterHeight:0,
+        cssPrefix:"reporttable",
+        pollInterval:400,
+        resizePollInterval:2000,
+        headerCell:"th",
+        scrollContainer:null,
+        scrollDefaultObject:null
```
Comment (kyrozetera) [inline dist/reporttable.js]: Should we name this option `scrollEventObject`, or something, so the purpose is clearer from the name?

**Review by kyrozetera — CHANGES_REQUESTED** [review]
Just a couple of small things, otherwise looks good.

**Comment by peterbrescia** [comment]
Not sure how original minification was achieved, but we should probably figure out how and update the min.js and map files too.

**Comment by kyrozetera** [comment]
@peterbrescia Good catch. We don't currently use the `min` file, but we could swap that out in Koddi also.

**Comment by peterbrescia** [comment]
Would be good to get this out. Looks like review has been addressed?

**Review by kyrozetera — APPROVED** [review]

**Review by peterbrescia — APPROVED** [review]

**Comment by kyrozetera** [comment]
@peterbrescia Yes, we can merge this one now, and we'll need to create a PR to update to this repo in koddi.

**Pull request merged**